lib.js

```diff
@@ -0,0 +1,28 @@
+function computeTitle(title) {
+  const getting = browser.storage.sync.get({
+    prefix: '',
+    prefixGlue: '',
+  });
+
+  return getting.then((result) => {
+    const computedTitle = result.prefix
```

Comment:
String interpolation here would make it more readable

Reply:
ok